code/modules/ruins/objects_and_mobs/ash_walker_den.dm

```diff
@@ -7,8 +7,8 @@
 	icon_state = "ash_walker_nest"
 	icon_living = "ash_walker_nest"
 	faction = list("ashwalker")
-	health = 200
-	maxHealth = 200
+	health = 400
+	maxHealth = 400
```

Comment:
It does after watching a goliath finish one off in the space of a minute. Once the nest is destroyed, there's no way to create more eggs. That's it for the Ash Walkers.